rpc/client/client.go

```diff
@@ -36,6 +36,14 @@ const (
 	subscribeTimeout = 5 * time.Second
 )
 
+type ValidationStatus int
+
+const (
+	BlockNotValidated = iota
+	BlockValidated
+	BlockValidatedWithSL
+)
```

Comment:
imo better to stick to p2p as `BlockValidated` could be amibigous. 

```suggestion
type BlockValidationStatus int

const (
	NotValidated = iota
	P2PValidated
	SLValidated
)
```